java/common/exception/ErrorMessage.java

```diff
@@ -17,12 +17,51 @@
  * under the License.
  */
 
-package com.vaticle.typedb.driver.common.exception;
+package com.typedb.driver.common.exception;
 
-public abstract class ErrorMessage extends com.vaticle.typedb.common.exception.ErrorMessage {
+import java.util.HashMap;
+import java.util.Map;
 
-    private ErrorMessage(String codePrefix, int codeNumber, String messagePrefix, String messageBody) {
-        super(codePrefix, codeNumber, messagePrefix, messageBody);
+public abstract class ErrorMessage {
+    private static Map<String, Map<Integer, ErrorMessage>> errors = new HashMap<>();
+    private static int maxCodeNumber = 0;
+    private static int maxCodeDigits = 0;
+
+    private final String codePrefix;
+    private final int codeNumber;
+    private final String message;
+    private String code = null;
+
+    protected ErrorMessage(String codePrefix, int codeNumber, String messagePrefix, String messageBody) {
```

Comment:
It's `protected` so we could extend it in `Console`. Doesn't sound like a very bad thing exposed to other users. But might make it `private` and copypaste the class into `Console`!